src/main/resources/application.properties

```diff
@@ -14,6 +18,21 @@ mp.messaging.incoming.ingress.group.id=integrations
 mp.messaging.incoming.ingress.key.deserializer=org.apache.kafka.common.serialization.StringDeserializer
 mp.messaging.incoming.ingress.value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
 
+# Output queue to Camel (notifications-sender)
+mp.messaging.outgoing.toCamel.connector=smallrye-kafka
+mp.messaging.outgoing.toCamel.topic=notifs
+mp.messaging.outgoing.toCamel.group.id=integrations
+mp.messaging.outgoing.toCamel.key.serializer=org.apache.kafka.common.serialization.StringSerializer
+mp.messaging.outgoing.toCamel.value.serializer=org.apache.kafka.common.serialization.StringSerializer
+
+# Input queue from camel sender
+mp.messaging.incoming.fromCamel.connector=smallrye-kafka
+mp.messaging.incoming.fromCamel.topic=notif-return
```

Comment:
What would happen if we deploy this on the servers and we don't yet have these topics?  It would fail?

Reply:
Both channels should probably be disabled until the real topics are available:

```
mp.messaging.outgoing.toCamel.enabled=false
mp.messaging.incoming.fromCamel.enabled=false
```

See https://smallrye.io/smallrye-reactive-messaging/smallrye-reactive-messaging/2.5/advanced/advanced.html#disabling-channels

Reply:
I disabled the channels.